frontend/magic_links/templates/invalid.html

```diff
@@ -4,23 +4,28 @@
 
 {% block content %}
   <div>
-      <h1 class="govuk-heading-xl">Link expired</h1>
+      {# TODO: Figure out translation for Link expired #}
+      <h1 class="govuk-heading-xl">{% trans %} Link expired {% endtrans %}</h1>
   </div>
 <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" data-module="govuk-error-summary">
   <h2 class="govuk-error-summary__title" id="error-summary-title">
-      This link has expired
+      {# TODO: Figure out which should be used. #}
+      {# This link has expired #}
+      {% trans %} Magic link expired {% endtrans %}
```

Comment:
`Magic link expired` was the closest translation I could find... so I replaced it and translated it.  I'll also raise this as I'm uncertain if this should've actually been replaced.